omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -846,7 +847,7 @@
 default['private_chef']['dark_launch']['sql_users'] = true
 default['private_chef']['dark_launch']['add_type_and_bag_to_items'] = true
 default['private_chef']['dark_launch']['reporting'] = true
-default['private_chef']['dark_launch']['actions'] = true
+default['private_chef']['dark_launch']['actions'] = false
```

Comment:
This is minor but potentially - customers will start getting 404 from that endpoint IIRC, is that going to be a surprise for anyone? Will it cause any problems with data feeds into automate? (I'm not sure if these come through `actions` or a different path.)

Reply:
As far as I am aware, the /actions endpoint was not being used for anyting other than actions and analytics. But I will try to look for more evidence that, that is actually true.